src/test/ui/anon-params-denied-2018.rs

```diff
@@ -1,4 +1,6 @@
-// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// Test that anonymous parameters are disallowed in 2018 edition.
```

Comment:
I think the copyright notice has to be the first thing in the file.

Reply:
Ah, thanks. I didn't notice I had done that.